assimilate.csh

```diff
@@ -88,6 +88,14 @@ else if ($?PBS_NODEFILE) then
    echo "tasks_per_node : $NUMTASKS_PERNODE"
    echo " "
 
+   # KDR; debugging differences between 2019-09-30-21600 
+   # before and after cheyenne July down time.
+   # Mick says the old MPT, which was used to build and run the old filter,
+   # was mpt/2.21.
+   module load mpt/2.21
```

Comment:
Do you have an already loaded mpt? If so, shouldn't a "module swap" be more suitable here rather than "module load"?